### DISCUSSION TITLE
The content

### DISCUSSION THREAD

**Discussion by Feofilakt** [statiqdev/Discussions#21] [Q&A] [answered]
Hello! Thank you for the theme example, but where can I found a content for it?

**Comment by daveaglick** [comment, marked as answer]
@Feofilakt The theme provides things like styling, navigation, archives and tag indexes, etc. Generally you'll install the theme ([instructions here](https://github.com/statiqdev/CleanBlog#installation)) and then write Markdown files in the `input/posts` folder.

The best example I can point to at the moment of a full blog using the theme is my own: https://github.com/daveaglick/daveaglick - feel free to clone it as an example and then start from there.

Does that help answer your question?

**Reply by Feofilakt** [reply]
Yes, thank you!
